script/Arcueid/Day 5/05_00_ARC05_5.txt

```diff
@@ -179,12 +179,12 @@ Wait, she just said something completely outrageous―――!
 [sha:afcb5e95b1f7aacac5f5f37f6ba75d1d0723e9d6]{
 -- Page 383, Offset 9600.
 -- 「う、」
-"%{i}Wah%{/i}."
+"%{i}Wah%{/i}――"
 }
 [sha:be31f004f5600d4de80d1bb0838a5ba7a275e37e]{
 -- Page 384, Offset 9601.
 -- 「うう、」
-"%{i}Waah%{/i}."
+"%{i}Waah%{/i}――"
```

Comment:
Maybe have this be 2 CJK dashes so they lengthen with the waah. Or just have them all be 3